bundle.go

```diff
@@ -321,10 +321,10 @@ func (w *BundleDownloader) Start() {
 				if _, ok := err.(*timeoutError); ok {
 					continue
 				}
-				go func() {
-					req.backoffFunc()
-					w.bm.enqueueRequest(req)
-				}()
+				go func(r *DownloadRequest, bm *bundleManager) {
+					r.backoffFunc()
+					bm.enqueueRequest(r)
```

Comment:
Not yet. However, the race is in tests but not code. There's enough sleep time to avoid it from actually happening. To resolve that race requires more code change.